internal/pkg/util/env/create.go

```diff
@@ -103,7 +103,7 @@ func warnDeprecatedEnvUsage(hostEnvs []string) {
 				if key != "" {
 					legacyEnv := LegacySingularityEnvPrefix + key
 					newEnv := ApptainerEnvPrefix + key
-					sylog.Warningf("DEPRECATED USAGE: Environment variable %s will not be supported in the future, use %s instead", legacyEnv, newEnv)
+					sylog.Infof("INFO: Environment variable %v is set, but %v is preferred", legacyEnv, newEnv)
```

Comment:
Don't include `INFO:` in the message; sylog.Infof will print that itself.  Likewise in the other cases.

Reply:
updated, thanks for the comments.